languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -25165,6 +25165,22 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
     <!-- Groß- und Kleinschreibung -->
     <!-- ====================================================================== -->
     <category id="CASING" name="Groß-/Kleinschreibung">
+        <rule id="ZU_HEULEN_VS_ZUM_HEULEN" name="zu heulen ist zum Heulen">
+            <pattern>
+                <token skip="-1" min="0"/>
+                <token inflected="yes" skip="-1">sein<exception postag="ADJ.*" postag_regexp="yes" scope="next"></exception></token>
+                <marker>
+                    <token>zu</token>
+                    <token>heulen</token>
```

Comment:
Diese Regel ist ultraspeziell – sie schlägt nur in diesem einen Fall an. Geht da nicht noch mehr? Zum Lachen, Weinen, Schreien, Verzweifeln...